compiler/llvm/clangUtil.cpp

```diff
@@ -2024,6 +2024,21 @@ void runClang(const char* just_parse_filename) {
     }
   }
 
+  // Remove -DCHPL_DEBUG, -DCHPL_OPTIMIZE, -DNDEBUG from the args
```

Comment:
It might be worth mentioning in the comment why we're removing them, that they're the settings from the runtime build and we'll be using the appropriate ones from the compilation instead.